Contributing.md

```diff
@@ -83,3 +147,4 @@ not to add this new member. Members who do not participate consistently at the l
 a majority of the other members are expected to resign.
 
 
+[Dashboard]: https://expressjs.github.io/statusboard/projects
```

Comment:
I made the suggestion above, but I would prefer to keep the link in-line.  It makes it much easier to find if it changes or if you are reviewing changes.

Reply:
done, removed it and made it inline!